src/v2/ion/RemediationConstants.js

```diff
@@ -42,6 +42,68 @@ const FORMS = {
   TERMINAL: 'terminal',
 };
 
+const FORMNAME_CLASSNAME_MAPPING = {
+  'identify': ['identify', 'primary-auth'],
+  'identify-password': ['identify', 'primary-auth'],
+  'select-identify': ['select-identify'],
+  'enroll-profile': ['enroll-profile', 'registration'],
+
+  // 1st step of the challenge flow for sms/call
+  'authenticator-verification-data-sms': ['authenticator-verification-data-sms'],
+  'authenticator-verification-data-phone': ['authenticator-verification-data-phone'],
+
+  'challenge-authenticator-voice': ['challenge-authenticator-voice', 'mfa-verify-passcode'],
+  'challenge-authenticator-sms': ['challenge-authenticator-sms', 'mfa-verify-passcode'],
+  'challenge-authenticator-email': ['challenge-authenticator-email', 'mfa-verify-passcode'],
+  'challenge-authenticator-password': ['challenge-authenticator-password', 'mfa-verify-password'],
+  'challenge-authenticator-security_question': ['challenge-authenticator-security_question', 'mfa-verify-question'],
+
+  'enroll-authenticator-voice': ['enroll-authenticator-voice', 'enroll-call'],
+  'enroll-authenticator-sms': ['enroll-authenticator-sms', 'enroll-sms'],
+  'enroll-authenticator-email': ['enroll-authenticator-email', 'enroll-email'],
+  'enroll-authenticator-password': ['enroll-authenticator-password', 'enroll-password'],
+  'enroll-authenticator-security_question': ['enroll-authenticator-security_question', 'enroll-question'],
+
+  'select-authenticator-authenticate': ['select-authenticator-authenticate'],
+  'select-authenticator-enroll': ['select-authenticator-enroll', 'enroll-choices'],
+
+  'select-authenticator-enroll-data': ['select-authenticator-enroll-data'],
+  'authenticator-enrollment-data': ['authenticator-enrollment-data'],
+
+  'reenroll-authenticator': ['reenroll-authenticator', 'password-expired'],
+  'reenroll-authenticator-warning': ['reenroll-authenticator-warning'],
+  'reset-authenticator': ['reset-authenticator'],
+  'skip': ['skip'],
+
+  'success-redirect': ['success-redirect'],
+  'redirect-idp': ['redirect-idp'],
+
+  'cancel-transaction': ['cancel-transaction'],
+};
+
+const getClassNameMapping = (appState) => {
+  const formName = appState.get('currentFormName');
+  let authenticatorType = appState.get('authenticatorType');
+  const recoveryAuthenticator = appState.get('recoveryAuthenticator');
+  let isPasswordRecoveryFlow = recoveryAuthenticator && recoveryAuthenticator.type === 'password';
+
+  if (authenticatorType === 'phone') {
+    // both sms and call have same type
+    authenticatorType = appState.get('currentAuthenticatorEnrollment') &&
+    appState.get('currentAuthenticatorEnrollment').methods[0].type;
+  }
+  let key = `${formName}`;
+  if (authenticatorType) {
+    key += `-${authenticatorType}`;
+  }
+
+  if (isPasswordRecoveryFlow) {
+    // password reset flow from identifier page with recoveryAuthenticator add forgot-password class
+    FORMNAME_CLASSNAME_MAPPING[key] = ['identify', 'forgot-password'];
```

Comment:
For my information, what `key`s would this override in Password Recovery Flow?

Reply:
We have 2 ways to start password recover in M1 

- From Identifier screen when we have both username and password shown on identifier page
- Or from password authenticator challenge screen 

the first flow is similar to the V1 password reset flow, hence we are adding the V1 class name to that screen.

Reply:
I think we may have another recovery flow where we get `recoveryAuthenticator ` in the response 

<img width="469" alt="Screen Shot 2020-07-21 at 2 01 26 PM" src="https://user-images.githubusercontent.com/23267876/88107069-582c9280-cb5b-11ea-9a1f-4110e23b1786.png">